op-acceptor/logging/filelogger.go

```diff
@@ -316,6 +316,18 @@ func (l *FileLogger) GetAllLogsFileForRunID(runID string) (string, error) {
 	return filepath.Join(baseDir, "all.log"), nil
 }
 
+// GetRawEventsFileForRunID returns the path to the raw_go_events.log file for the given runID
+func (l *FileLogger) GetRawEventsFileForRunID(runID string) (string, error) {
+	if runID == "" {
+		return "", fmt.Errorf("runID is required")
+	}
+	baseDir, err := l.GetDirectoryForRunID(runID)
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(baseDir, "raw_go_events.log"), nil
+}
```

Comment:
looks like several methods here are basically identical besides the file name, worth extracting to a shared function?